tests/alembic/conftest.py

```
@@ -0,0 +1,107 @@
+    )
+    cfg.set_main_option("sqlalchemy.url", alembic_database)
+    return cfg
+
+
+@pytest.fixture(scope="session")
+def alembic_database() -> Generator[str, None, None]:
+    """Provision a dedicated DB for Alembic verification tests."""
+
+    assert ALEMBIC_TEST_DB_URL.database == "alembic_migration_tests", (
+        "Can't set up Alembic test database fixture. "
+        + "ALEMBIC_TEST_DB_URL.database is {ALEMBIC_TEST_DB_URL.database}, "
```

Ruff (0.14.1)

Explicitly concatenated string should be implicitly concatenated

Remove redundant '+' operator to implicitly concatenate

(ISC003)